src/Login.cpp

```diff
@@ -66,6 +66,11 @@ std::shared_ptr<User> Login::userInput() {
                 createUser(name, password);
                 break;
             }
+            case 'q':
+            case 'Q': {
+                printf("Exiting InventoryManager.\n");
```

Comment:
This is the best print statement I've ever seen -- great job!

Reply:
<3